enterprise/server/util/oci/oci.go

```diff
@@ -220,7 +229,8 @@ func (c Credentials) Equals(o Credentials) bool {
 type Resolver struct {
 	env environment.Env
 
-	allowedPrivateIPs []*net.IPNet
+	imageTagToDigestLRU *expirable.LRU[string, string]
```

Comment:
we have our own lru in the `lru` package, I don't think there's a reason to depend directly on a third-party library for this. we typically handle expiration by storing the expiration time as part of the LRU entry.